lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -314,6 +317,18 @@
                 .attr('aria-disabled', !isEnabled);
         },
 
+        setSubmitButtonFocused: function( ){
+            $( this.submitButton )
+                .trigger('focus');
+            $('#submit').attr('title', gettext( 'Photo Captured successfully '));
+        },
+
+        capture_sound: function(){
+            var audio = document.createElement('audio');
+            audio.src = '/static/audio/camera_capture.wav';
+            audio.play();
```

Comment:
@AlasdairSwan Should I add this in DOM Element and then remove it or it could be used just like that. For reference http://stackoverflow.com/questions/1847220/javascript-document-createelement-delete-domelement